.upgrade.yml

```diff
@@ -0,0 +1,3 @@
+mappings:
+  Color: Tractorcow\Colorpicker\Color
```

Comment:
Super minor detail, I know, but can `TractorCow` be capitalised this way? Like `SilverStripe`